front/app/containers/App/VerificationModal/index.tsx

```diff
@@ -2,9 +2,9 @@ import React, { useCallback, useState, useEffect } from 'react';
 
 // components
 import Modal from 'components/UI/Modal';
-import VerificationSuccess from './VerificationSuccess';
+import VerificationSteps from 'components/SignUpIn/SignUp/VerificationSignUpStep/VerificationSteps';
```

Comment:
This is another sign that SignUpIn and this modal should live in the same place. Normally, we would disallow these types of imports in the future.